src/main/java/pageObjects/HeaderMenu.java

```diff
@@ -0,0 +1,68 @@
+package pageObjects;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+
+public class HeaderMenu {
+
+    @FindBy(xpath = "//a[@class='bag__tobag']")
+    private WebElement shoppingCart;
+
+    @FindBy(xpath = "(//a[contains(text(),'Ro')])[1]")
+    private WebElement language_Ro;
+
+    @FindBy(xpath = "(//a[contains(text(),'Ru')])[1]")
+    private WebElement language_Ru;
+
+    @FindBy(xpath = "(//a[contains(text(),'En')])[1]")
+    private WebElement language_En;
+
```

Comment:
as the xpath is only changing the RO, RU, EN - please think to include as a param value -> language(Ro) which will add the Ro/Ru/En as value into the xpath

@FindBy(xpath = "(//a[contains(text(), value)])[1]")
    private WebElement language

Reply:
As it is sting value it would be look like 
@findby(xpath = "(//a[contains(text(),"+ value + ")])[1]")

Reply:
I don't think so, as annotation will be applied at same time as page initiation, but you can make one WebElement which will catch all language links, and then in step you will be able to create a continuation of xpath which will select by text that will come as an argument. Ask @oleaprijilevskaia  of help as i helped her at selenium task with this kind of problem. 
+ Please avoid xpath index `[1]` at the end